api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -217,8 +218,8 @@ private Patient createBasicPatient() {
 		// patient.removeAddress(pAddress);
 		
 		patient.setDeathDate(new Date());
-		// patient.setCauseOfDeath("air");
-		patient.setBirthdate(new Date());
+		patient.setCauseOfDeath(Context.getConceptService().getConcept(3));
+		patient.setBirthdate(new Date(0));
```

Comment:
Yes. Got the failure:
PatientServiceTest.shouldCreatePatientWithValidatedIdentifier:313 Patient creation should have failed with identifier 123-1